py3status/modules/xrandr.py

```diff
@@ -333,7 +326,7 @@ def _refresh_py3status(self):
         """
         Send a SIGUSR1 signal to py3status to force a bar refresh.
         """
-        call(shlex.split('killall -s USR1 py3status'))
+        self.py3.command_run('killall -s USR1 py3status')
```

Comment:
If somebody have 3 monitors, they do this. 3 github calls, 3 spaceapi calls, etc. 👍